frontend/src/app/components/start/start.component.ts

```diff
@@ -50,4 +52,27 @@ export class StartComponent implements OnInit {
       });
   }
 
+  onMouseDown(event: MouseEvent, blockchainContainer: Element) {
+    this.mouseDragStartX = event.clientX;
+    this.blockchainScrollLeftInit = blockchainContainer.scrollLeft;
+    this.blockchainContainer = blockchainContainer;
```

Comment:
You can add `@ViewChild('blockchaincontainer') blockchainContainer: ElementRef;` to the class and you don't have to pass the element to the function and set it here.